tests/zfs-tests/cmd/libzfs_input_check.c

```diff
@@ -683,6 +683,16 @@ test_vdev_trim(const char *pool)
 	nvlist_free(required);
 }
 
+static void
+test_scrub(const char *pool)
+{
+	nvlist_t *required = fnvlist_alloc();
+	fnvlist_add_uint64(required, "scan_type", 1ULL << 30);
+	fnvlist_add_uint64(required, "scan_command", 1ULL << 31);
```

Comment:
What are these bit shifted values for?  Should they be in a macro?

Reply:
These are invalid values to assert if the libzfs_input check fails with EINVAL.

Reply:
This is a check included in the original PR to assert if `test_scrub()` fails with invalid input values. For `scan_type` I used 5 (0 to 4 are defined in `typedef enum pool_scan_func`), and for `scan_command` I used 3 (0 to 2 are defined in `typedef enum pool_scrub_cmd`).

I am also fine with completely removing this check.

Reply:
Probably overkill, but this builds for me:
```diff
diff --git a/tests/zfs-tests/cmd/libzfs_input_check.c b/tests/zfs-tests/cmd/libzfs_input_check.c
index f62b7dbd7..eaeb2ff7e 100644
--- a/tests/zfs-tests/cmd/libzfs_input_check.c
+++ b/tests/zfs-tests/cmd/libzfs_input_check.c
@@ -27,6 +27,7 @@
 #include <sys/vdev_impl.h>
 #include <sys/zfs_ioctl.h>
 #include <sys/zfs_bootenv.h>
+#include <sys/fs/zfs.h>
 
 /*
  * Test the nvpair inputs for the non-legacy zfs ioctl commands.
@@ -688,12 +689,13 @@ test_vdev_trim(const char *pool)
        nvlist_free(required);
 }
 
+/* Test with invalid values */
 static void
 test_scrub(const char *pool)
 {
        nvlist_t *required = fnvlist_alloc();
-       fnvlist_add_uint64(required, "scan_type", 5);
-       fnvlist_add_uint64(required, "scan_command", 3);
+       fnvlist_add_uint64(required, "scan_type", POOL_SCAN_FUNCS + 1);
+       fnvlist_add_uint64(required, "scan_command", POOL_SCRUB_FLAGS_END + 1);
        IOC_INPUT_TEST(ZFS_IOC_POOL_SCRUB, pool, required, NULL, EINVAL);
        nvlist_free(required);
 }
```

Reply:
Sure! I will modify this.